vars/junit.groovy

```diff
@@ -0,0 +1,29 @@
+// vars/junit.groovy
+
+/**
+ * run.groovy
+ *
+ * Wrapper for junit step to allow for skipped tests.
+ *
+ */
+
+def call(String testResults = '') {
```

Comment:
Shouldn't this accept a `config` _Map_ also?

Reply:
It should, but I could not figure out how to make that work and also accept the String parameter, and we currently have no calls using a config Map.